cirq-core/cirq/sim/clifford/clifford_tableau.py

```diff
@@ -17,10 +17,12 @@
 
 import cirq
 from cirq import protocols
+from cirq._compat import deprecated_class
 from cirq.ops.dense_pauli_string import DensePauliString
 from cirq.value import big_endian_int_to_digits
 
 
+@deprecated_class(deadline='v0.12', fix='Import CliffordTableau from cirq or cirq.qis instead.')
 class CliffordTableau:
     """Tableau representation of a stabilizer state
     (based on Aaronson and Gottesman 2006).
```

Comment:
You should be able to remove the implementation and just make it equal to cirq.qis.CliffordTableau.

Reply:
Done.